This is a temporary patch for an upstream issue:

Minor issue in building qpdf on macOS #1424
https://github.com/qpdf/qpdf/issues/1424

--- a/libqpdf/QPDFJob.cc
+++ b/libqpdf/QPDFJob.cc	2025-05-02 17:14:06.046642993 +0200
@@ -123,7 +123,7 @@
         // Recompress existing jpeg.
         decode_level = qpdf_dl_all;
         config = Pl_DCT::make_compress_config(
-            [quality](jpeg_compress_struct* cinfo) { jpeg_set_quality(cinfo, quality, false); });
+            [quality](jpeg_compress_struct* cinfo) { jpeg_set_quality(cinfo, quality, FALSE); });
     }
 }
 
